docs/providers/aws/guide/serverless.yml.md

```diff
@@ -44,9 +44,10 @@ provider:
   logRetentionInDays: 14 # Set the default RetentionInDays for a CloudWatch LogGroup
   kmsKeyArn: arn:aws:kms:us-east-1:XXXXXX:key/some-hash # KMS key arn which will be used for encryption for all functions
   deploymentBucket:
+    blockPublicAccess: true # Prevents public access via ACLs or bucket policies. Default is false
+    disableDefaultBucketPolicyCreation: true # Prevents creation of default bucket policy when framework creates the deployment bucket. Default is false
```

Comment:
Ideally we should list default values (although I know our listing is not perfectly compliant with this).

So maybe let's change it to `false`